Dockerfile

```diff
@@ -1,16 +1,22 @@
-FROM ubuntu:20.04
+FROM python:3.9.6-buster
 
 #MAINTANER Your Name "youremail@domain.tld"
 
 RUN apt-get update -y && \
-    apt-get install -y python3-pip python-dev
+    pip install pipenv && \
+    git clone https://github.com/cyfronet-fid/oms-adapter-jira.git && \
+    cd oms-adapter-jira && \
+    pipenv install --dev
```

Comment:
We don't actually need dev dependencies. Also, please add `--system` option to avoid virtualenv creation in a container